judge/models/comment.py

```diff
@@ -151,3 +151,16 @@ class Meta:
         unique_together = ['voter', 'comment']
         verbose_name = _('comment vote')
         verbose_name_plural = _('comment votes')
+
+class CommentLock(models.Model):
+    page = models.CharField(max_length=30, verbose_name=_('associated Page'), db_index=True,
```

Comment:
`verbose_name` should be all lowercase, I think.

Reply:
I will change this along with #747 for the sake of both models having the same definition in a separate PR.